unit_tests.py

```diff
@@ -20,67 +19,100 @@ def test_remove_duplicates(self):
     def test_get_indexes_to_remove(self):
         # define dataframe for input
         test_df = pd.DataFrame({"submission":pd.Series(["1", "2", "2", "2", "1", "3"], dtype="object"),
-                                "pcMapped":pd.Series([0.1, 0.2, 0.3, 0.4, 0.1, 0.6], dtype=float)})
+                                "pcMapped":pd.Series([0.1, 0.2, 0.3, 0.4, 0.2, 0.6], dtype=float)})
```

Comment:
Just checking that 5th entry for this is meant to be `0.2`?  In lines 15 and 28 its `0.1`, but otherwise the dataframes are the same.